### PR TITLE
Fixing Razor Cursor (which was broken on Windows & webkit backends)

### PR DESCRIPTION
**Pull request by jonoomph**
Fixing razor cursor, which was broken on Windows (i.e. webkit backend) when we moved from PNG to SVG icon file. Also, adding a new way to target CSS for webkit VS webengine.

We will eventually move away from a JS/CSS approach to the timeline, so this is really a bandage for now, to allow some slight differences on how we CSS style the timeline (depending on the webkit / webengine backend used).

**Testing Instructions:**
- Launch this build on Windows (or run with `--web-backend webkit` backend)
- Drop an emoji or video on the timeline
- Click the 'Razor' toolbar icon
- Move your mouse over the clip, and you should see the razor cursor

### CODE REVIEW

File: src/timeline/js/mixin_webengine.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ function init_mixin() {
     angular.element(document).ready(function () {
       timeline.qt_log("INFO", "Angular Ready");
       $("body").scope().enableQt();
+      $("body").addClass("webengine");
```
Comment (jonoomph) [inline src/timeline/js/mixin_webengine.js:42]: Adding a custom class to the `<body class="webengine">`, so we can use it to target the difference backends using CSS.

File: src/timeline/media/css/main.css [hunk 2]
```diff
@@ -303,7 +303,11 @@ img {
   margin-right: 1px;
 }
 
-.razor_cursor {
+.webkit .razor_cursor {
+  cursor: url(../images/razor_line_with_razor.png), default !important;
```
Comment (jonoomph) [inline src/timeline/media/css/main.css:307]: Apparently webkit does not support SVG files for CSS cursors. I tried a bunch of different test files, different sizes, and never got one to appear. So using a PNG instead (but only for webkit).

Comment (JacksonRG) [reply src/timeline/media/css/main.css:307]: Bizarre! But good to know!

**Review by JacksonRG — APPROVED** [review]
Merge away!

**Pull request merged**